tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -375,6 +377,12 @@ private:
    RNTupleSerializer::StreamerInfoMap_t fStreamerInfos;
 
 protected:
+   /// Set of optional features supported by the persistent sink
+   struct RFeatures {
```

Comment:
Are we foreseeing more features?

Reply:
Good question. Nothing immediately, but I could imagine more flags in the future, for instance if the backend already does compression.